server/core/src/miscServerFunct.cpp

```diff
@@ -2983,12 +2982,24 @@ irods::error get_script_output_single_line(
     const std::string&              script_name,
     const std::vector<std::string>& args,
     std::string&                    output ) {
-    bp::ipstream output_stream;
-    bp::child c(script_language, args, bp::std_out > output_stream);
-    output.clear();
-    std::getline(output_stream, output);
-    c.wait();
-    return SUCCESS();
+    namespace bp = boost::process;
+    try {
+        auto tmp = args;
+        tmp.insert(tmp.begin(), fmt::format("{}/scripts/{}", irods::get_irods_home_directory().string(), script_name));
+        bp::ipstream output_stream;
+        bp::child c(script_language, tmp, bp::std_out > output_stream);
```

Comment:
We need to consider stderr and what that means to this function.

Reply:
Well, the old version of the function doesn't do anything with stderr, so it's new territory if we want to do that.

Reply:
True, but we want to make sure things run and operate as expected. In 99% of the cases, we expect stderr to be empty. However, if it's not, we need to check it (and log it).

As for whether stderr should result in this function returning an error, that is something that needs to be discussed as a group. That also means we need to know what scripts are being invoked via this function. Knowing that will help us decide on what to do when stderr contains information.